org.kde.kwave.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "org.kde.kwave",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kwave",
+    "rename-icon": "kwave",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        "shared-modules/libmad/libmad.json",
+        {
+            "name": "audiofile",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+                    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+                },
+                {
+                    "type": "patch",
+                    "path": "audiofile-gcc6.patch"
+                }
+            ]
+        },
+        {
+            "name": "id3lib",
+            "buildsystem": "simple",
+            "build-commands": [
```

Comment:
you should use autotools and use a `script` with dest-filename `autogen.sh` source to do the autoconf stuff.

Reply:
That doesn't work for me. Using `shell` source works though.